apps/web/components/LocaleSearchItem/LocaleSearchItem.tsx

```
@@ -0,0 +1,27 @@
+
+interface ListProps {
+  query: string;
+}
+
+interface ItemProps {
+  item: LocalItem;
+}
+
+export function LocaleSearchItemList({query}: ListProps) {
+  const {data} = useRequestGetLocalSearch(query);
+  return <VStack gap={4}>{data?.items.map((item: LocalItem) => <LocaleSearchItem item={item} />)}</VStack>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)